docs/client/concepts/parameter-stores.mdx

```diff
@@ -7,30 +7,95 @@ slug: /client/concepts/parameter-stores
 Only available for Statsig Mobile Client SDKs
 :::
 
-Parameter Stores are a new way of organizing and managing parameters in your mobile app via the Statsig console.  They are now available for Statsig Android, iOS, and React Native sdks.
+### **Parameter Stores: Simplifying Your Mobile App Configuration Management**
```

Comment:
I updated the page just now - its for client sdks, not just mobile.  youll have to rebase